.github/dependabot.yml

```diff
@@ -14,54 +12,44 @@ updates:
   directory: "/"
   schedule:
     interval: weekly
-  ignore:
-    - dependency-name: "wp-phpunit/wp-phpunit"
```

Comment:
Why are we removing this?

Reply:
Oh! This is done with the automated scripts, which assumes all the Dependabot files are the same. You edit one and copy it over the others. 
I'm not sure why this one is different. I will investigate.

Reply:
It was me back in August! https://github.com/humanmade/altis-dev-tools/pull/601

I recall this was to stop dependabot from trying to update `wp-phpunit` because we need it to stick to the same exact WP version.
It feels like a less painful solution would be to add the same exclusion to all modules, rather than start adding exceptions or manuals steps to the release process.

Reply:
Meanwhile, I've replaced the missing extra ignore settings.